src/TLuaInterpreter.cpp

```diff
@@ -2742,61 +2682,55 @@ int TLuaInterpreter::disableCommandLine(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#replace
 int TLuaInterpreter::replace(lua_State* L)
 {
-    std::string a1 = "";
-    std::string a2 = "";
     int n = lua_gettop(L);
     int s = 1;
     if (!lua_isstring(L, s)) {
         lua_pushstring(L, "replace: wrong argument type");
         lua_error(L);
         return 1;
-    } else {
-        a1 = lua_tostring(L, s);
-        s++;
     }
+    QString text = lua_tostring(L, s);
```

Comment:
Need `QString::fromUtf8(...)` here...

Reply:
... actually you will need this in *quite a few* places!

Reply:
Actually, no:

![image](https://user-images.githubusercontent.com/110988/95015335-914dac80-064c-11eb-88e6-2c5328334dac.png)

As @smurfix pointed out, we are needlessly using it everywhere, because Qt already uses internally: https://doc.qt.io/qt-5/qstring.html#QString-7